docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,13 @@ develop
     *    - Type
          - Change
 
+    *    - |improved| |fixed|
+         - Sweep progress is now persisted as a blob.
+           This allows us constant time loading of the persisted SweepResult which was previously linear in the size of the table being swept.
```

Comment:
@gbonik would like to check that this makes sense in dbkvs/oracle? This change should make us use `getLatestCellQuery` rather than `getLatestRowQuery` when connecting to the KVS. It looks like the GROUP BYs there would only have 0 or 1 group.

In C* I'm pretty sure this does indeed condense a linear-time operation into a constant-time one.